Services/Common/OJS.Services.Common.Models/Submissions/SubmissionExecutionResultServiceModel.cs

```diff
@@ -0,0 +1,23 @@
+﻿namespace OJS.Services.Common.Models.Submissions;
+
+using AutoMapper;
+using FluentExtensions.Extensions;
+using SoftUni.AutoMapper.Infrastructure.Models;
+
+public class SubmissionExecutionResultServiceModel : IMapExplicitly
+{
+    public int SubmissionId { get; set; }
+
+    public string? Exception { get; set; }
+
+    public string? ExecutionResult { get; set; }
+
+    public void RegisterMappings(IProfileExpression configuration)
+        => configuration.CreateMap<SubmissionExecutionResult, SubmissionExecutionResultServiceModel>()
```

Comment:
These 2 models are confusing. Maybe rename `SubmissionExecutionResultServiceModel` to `SerializedSubmissionExecutionResultServiceModel` so that the difference is more distinct.

Reply:
Done.